drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/utils/ConstantHolder.java

```diff
@@ -15,22 +15,29 @@
  */
 package org.drools.workbench.screens.scenariosimulation.client.utils;
 
+import static org.drools.scenariosimulation.api.utils.ConstantsHolder.MVEL_ESCAPE_SYMBOL;
+
 /**
  * Class used to store constants used throughout the code
  */
 public class ConstantHolder {
 
+    private ConstantHolder() {
+        // Not instantiable
+    }
+
     public static final String FA_ANGLE_DOWN = "fa-angle-down";
     public static final String FA_ANGLE_RIGHT = "fa-angle-right";
     public static final String HIDDEN = "hidden";
     public static final String NODE_HIDDEN = "node-hidden";
     public static final String VALUE = "value";
-
     public static final String SAVE = "Save";
 
+    public static final String EXPRESSION_INSTANCE_PLACEHOLDER = "Expression </>";
```

Comment:
in prototype I see "expression </>" not "Expression </>"

Reply:
@dupliaka  done!